docs/contributing/code-style.md

```diff
@@ -152,41 +152,40 @@ all code via our [linter tool](../testing/linters.md) with
 `tools/lint --only=prettier --fix`. You can also [integrate it with your
 editor](https://prettier.io/docs/en/editors.html).
 
-Combine adjacent on-ready functions, if they are logically related.
-
-The best way to build complicated DOM elements is a Handlebars template
+- Combine adjacent on-ready functions, if they are logically related.
+- The best way to build complicated DOM elements is a Handlebars template
 like `web/templates/message_reactions.hbs`. For simpler things
-you can use jQuery DOM building APIs like so:
-
-```js
-var new_tr = $('<tr />').attr('id', object.id);
-```
-
-Passing a HTML string to jQuery is fine for simple hardcoded things
-that don't need internationalization:
+you can use jQuery DOM-building APIs like this:
+  ```js
+  const $new_tr = $('<tr />').attr('id', object.id);
+  ```
+- Passing a HTML string to jQuery is fine for simple hardcoded things
+that don't need internationalization (but avoid programmatically building
+complicated strings):
+  ```js
+  foo.append('<p id="selected">/</p>');
+  ```
 
-```js
-foo.append('<p id="selected">/</p>');
-```
 
-but avoid programmatically building complicated strings.
+### Attach behaviors to event listeners
 
-We used to favor attaching behaviors in templates like so:
+Attach callback functions to events using jQuery code. For example:
 
 ```js
-<p onclick="select_zerver({{id}})">
+$("#main_div").on("click", ".message_reaction", function (e) {
```

Comment:
This example is a bit weird, in that these days, we mainly do `$("body")` as the container for this sort of thing.